docs/pages/desktop-access/getting-started.mdx

```diff
@@ -444,13 +444,36 @@ the filepath to the `der_ca_file` configuration variable.
   order to resolve this.
 </Admonition>
 
+Install Teleport on the host where you will run the Teleport Deskop Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
 In order to enable Desktop Access in Teleport, add the following section in
-`teleport.yaml` on your Linux server. For a detailed description of these
+`/etc/teleport.yaml` on your Linux server. For a detailed description of these
 configuration fields, see the
 [configuration reference](./reference/configuration.mdx) page.
 
-<Details title="Desktop Service Configuration" min="8.0" scopeOnly={true} scope={["oss", "enterprise"]}>
+<Tabs>
+<TabItem scope={["oss","enterprise"]} label="Self-Hosted">
+
+For Teleport Self-Hosted, Windows Desktop Service can connect to the auth service with a
```

Comment:
It's more likely that self-hosted will also use the proxy's reverse tunnel.

Reply:
I have changed this to reverse tunnel @zmb3